scripts/local/run_stop.sh

```diff
@@ -5,26 +5,13 @@
 set -e # Stop on first error
 source ./scripts/utils.sh
 
-if [[ $# -gt 1 ]]; then
+if [[ $# -gt 0 ]]; then
     echo "Invalid number of arguments. Usage:"
-    echo "   ./scripts/local/run_stop.sh             # stop the running containers and preserve the network for subsequent runs"
-    echo "   ./scripts/local/run_stop.sh -c          # stop the running containers and clean the network"
+    echo "   ./scripts/local/run_stop.sh             # stop the running containers and clean the network"
     exit 1
 fi
 
-clean=false
-while getopts c flag
-do
-    case "${flag}" in
-        c ) clean=true;;
-    esac
-done
-
-# If clean flag is set we remove network running file to start the network from scratch
-if $clean
-then
-    echo "Cleaning network"
-    rm -f NETWORK_RUNNING
-fi
-
-docker compose -f docker/docker-compose-run.yml --project-directory ./ stop
\ No newline at end of file
+echo "Cleaning network"
+rm -f NETWORK_RUNNING
```

Comment:
Since we've removed `NETWORK_RUNNING`, this script just calls `docker compose stop`. How about we remove this script altogether and stop the containers from `run.sh` directly? I think that's the only remaining place we call `run_stop.sh`. Even without the specific call to `docker compose stop`, CTRL-C in the terminal that is running docker compose should stop all the containers.

Reply:
Additionally, I've oberved scenarios in which `avalanche network stop`, `avalanche network clean`, and `docker compose stop` leave the containers in an odd state for subsequent runs. The only reliable fix to this has been to remove the containers altogether.

Since creating the containers takes much less time than actually starting the network (which we do unconditionally), I think we could improve UX further by also unconditionally removing the containers. This can be done with `docker compose -f docker/docker-compose-run.yml --project-directory ./ rm -f`

Reply:
Yeah, I left `run_stop.sh` so a user could call it to kill `run.sh`, which is probably only useful if it's called as a detached process, and in that case, the user should know how to kill it anyway. I'll remove it.

Reply:
AFAICT, `--force-recreate` should accomplish the same thing, so I've added that to the script.

Reply:
I think that should work too 👍